src/Handler/Info/ObjectInfoHandler.php

```diff
@@ -0,0 +1,203 @@
+use Keboola\TableBackendUtils\Table\Bigquery\BigqueryTableReflection;
+use Keboola\TableBackendUtils\TableNotExistsReflectionException;
+
+final class ObjectInfoHandler implements DriverCommandHandlerInterface
+{
+    public GCPClientManager $clientManager;
+
+    public function __construct(GCPClientManager $clientManager)
+    {
+        $this->clientManager = $clientManager;
+    }
+
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     */
+    public function __invoke(
+        Message $credentials,
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof ObjectInfoCommand);
+
+        $bqClient = $this->clientManager->getBigQueryClient($credentials);
+
+        $path = ProtobufHelper::repeatedStringToArray($command->getPath());
+
+        assert(count($path) !== 0, 'Error empty path.');
+
+        $response = (new ObjectInfoResponse())
+            ->setPath($command->getPath())
+            ->setObjectType($command->getExpectedObjectType());
+
+        switch ($command->getExpectedObjectType()) {
+            // DATABASE === PROJECT in BQ
+            case ObjectType::DATABASE:
+                return $this->getDatabaseResponse($path, $bqClient, $response);
+            case ObjectType::SCHEMA:
+                return $this->getSchemaResponse($path, $bqClient, $response);
+            case ObjectType::VIEW:
+                return $this->getViewResponse($path, $bqClient, $response);
+            case ObjectType::TABLE:
+                return $this->getTableResponse($path, $response, $bqClient);
+            default:
+                throw new UnknownObjectException(ObjectType::name($command->getExpectedObjectType()));
+        }
+    }
+
+    /**
+     * @return Generator<int, ObjectInfo>
+     */
+    private function getChildSchemas(BigQueryClient $bqClient): Generator
+    {
+        $datasets = $bqClient->datasets();
+        /** @var Dataset $child */
+        foreach ($datasets as $child) {
+            yield (new ObjectInfo())
+                ->setObjectType(ObjectType::SCHEMA)
+                ->setObjectName($child->info()['datasetReference']['datasetId']);
+        }
+    }
+
+    /**
+     * @return Generator<int, ObjectInfo>
+     */
+    private function getChildObjectsForSchema(BigQueryClient $bqClient, string $databaseName): Generator
+    {
+        $ref = new BigquerySchemaReflection($bqClient, $databaseName);
+        $tables = $ref->getTablesNames();
+        foreach ($tables as $table) {
+            yield (new ObjectInfo())
+                ->setObjectType(ObjectType::TABLE)
+                ->setObjectName($table);
+        }
+        $views = $ref->getViewsNames();
+        foreach ($views as $view) {
+            yield (new ObjectInfo())
+                ->setObjectType(ObjectType::VIEW)
+                ->setObjectName($view);
+        }
```

Comment:
tu oproti TR chyba `yield from $this->getChildSchemas($db, $databaseName);` pretoze nema nejake dalsie child